docs/platforms/gitlab-com.md

```
@@ -5,78 +5,138 @@ sidebar_label: GitLab.com
-
-If your organization prefers to use an organization user, you can create a new user in GitLab and generate a Personal Access Token for that user, following [our recommendations](#recommendations).
+This guide will assist you in effectively integrating CodeRabbit with SaaS GitLab.
 
-:::note
+## GitLab Access Tokens
 
-Keep in mind that if you want to change the review user, you can either disable the organization user or add a new user via the CodeRabbit UI. However, this will require manually removing the previous user from the projects and associated webhooks. Afterward, you will need to reinstall the CodeRabbit app for each project.
+To enable CodeRabbit to interact with your GitLab repositories, an access token is required. This token grants the necessary permissions for interacting with the Merge Requests and Discussions APIs.
 
-:::
+    1.	Personal Access Token (PAT): You can generate a PAT using a service account. We recommend creating a dedicated service account, associating it with the appropriate base group, and generating the PAT from this account.
```

Markdownlint (0.37.0)

Column: 7
Hard tabs

(MD010, no-hard-tabs)